sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -94,24 +94,24 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit fcd3693d1b54ddad1fcbe9e2c749c01628739346
-  # Merge: c8a0d84b 5d51c9a1
+  set(UNIFIED_RUNTIME_REPO "https://github.com/0x12CC/unified-runtime.git")
+  # commit ebf873fb5996c9ddca32bbb7c9330d3ffe15473c
+  # Merge: 633ec408 8f375039
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Fri May 3 19:12:25 2024 +0100
-  #    Merge pull request #1513 from aarongreig/aaron/minorL0DeviceInfoFixes
-  #    Fix some minor bugs in L0 urDeviceGetInfo revealed by PI->UR port
-  set(UNIFIED_RUNTIME_TAG fcd3693d1b54ddad1fcbe9e2c749c01628739346)
+  # Date:   Thu May 2 10:56:55 2024 +0100
+  #    Merge pull request #1535 from przemektmalon/przemek/sampled-image-fetch
+  #    [Bindless][Exp] Add device queries for sampled image fetch
+  set(UNIFIED_RUNTIME_TAG d3d3f6e519e804d109117e99ce6d84d10dba9131)
```

Comment:
Since the UR change only touches the L0 adapter these don't need to change, only the `fetch_adapter_source(level_zero` tag below.